andela/suyabay
# #116859349 Fix button below username not showing

#### What does this PR do?
- After compiling sass to CSS, the buttons below users on the admin dashboard were absent
  <img width="1430" alt="screen shot 2016-04-04 at 11 26 34 am" src="https://cloud.githubusercontent.com/assets/8840187/14242113/310817ca-fa58-11e5-9100-1185aff44fa8.png">
#### Description of Task to be completed?
- Inspect CSS of the image card
- Change the height property of the image card
#### How should this be manually tested?
- Visit the users listing on the admin dashboard. You should see an `EDIT PROFILE` button below each user avatar
#### What are the relevant pivotal tracker stories?
#116859349
#### Screenshots (if appropriate)
- After changing the image-card height property to 50%, the users listing now looks like this
  <img width="1218" alt="screen shot 2016-04-04 at 11 21 59 am" src="https://cloud.githubusercontent.com/assets/8840187/14242016/8e8e749e-fa57-11e5-9079-0fdb5c09e250.png">
